lib/versioning/semver-cargo/index.js

```diff
@@ -0,0 +1,35 @@
+const semver = require('../semver');
+
+function cargo2npm(input) {
+  return input.replace(',', '||');
```

Comment:
Does a comma in cargo versioning mean “AND” or “OR”?

Reply:
It isn't specified explicitly in the reference, but it most likely means "OR". It looks like the following version requirement: ```1.2.0, > 1.4.0 <= 1.6.5``` would mean `1.2.0 OR > 1.4.0 AND <= 1.6.5`, because it works the same way for npm with `||` instead of `,` and because a space between two comparators means `AND` in the reference.

Also it wouldn't make much sense to have more than 2 terms in an `AND` expression for version requirements. For example: `> 1.2.0 AND >= 1.4.0 AND < 1.9.0` would be equivalent to `>= 1.4.0 AND < 1.9.0`.